plugins/qlcplus.js

```diff
@@ -17,12 +17,13 @@ module.exports.export = function exportQLCplus(library, options) {
 
     // make null references in channel list link to a "No function" channel
     for (const mode of fixture.modes) {
-      for (let i=0; i<mode.channels.length; i++) {
-        if (mode.channels[i] == null) {
-          mode.channels[i] = 'No Function';
+      for (let i = 0; i < mode.channels.length; i++) {
+        if (mode.channels[i] === null) {
+          const noFunctionChKey = `No Function ${mode.name} ${i}`;
```

Comment:
`mode.name` reduces the probability that there is already a channel with that key – if the fixture has a `No function` and a `No function 1` channel, then a `null` at index 1 would use the existing `No function 1` channel which can have unintended side effects.